src/scss/_functions.scss

```diff
@@ -148,9 +145,13 @@
 /// @param {Color} $background - the hex color of the background the text will appear on
 /// @param {Number} $opacity [100] - the opacity percentage the text color should appear at
 @function oColorsGetTextColor($background, $opacity: 90) {
+	@if ($background == null) {
```

Comment:
This used to throw an error but not a rubbish one. Instead of return null, could we continue to throw an error but with a useful message?